src/Fusion.Summary.Api/ConfigurationExtensions.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Azure.Identity;
+
+namespace Fusion.Summary.Api;
+
+public static class ConfigurationExtensions
+{
+    public static void AddKeyVault(this WebApplicationBuilder builder)
+    {
+        var configuration = builder.Configuration;
+        var clientId = configuration["AzureAd:ClientId"];
+        var tenantId = configuration["AzureAd:TenantId"];
+        var clientSecret = configuration["AzureAd:ClientSecret"];
+        var keyVaultUrl = configuration["KEYVAULT_URL"];
+
+        if (!string.IsNullOrEmpty(keyVaultUrl))
+        {
+            Console.WriteLine($"Adding key vault using url: '{keyVaultUrl}', client id '{clientId}' and client secret {(string.IsNullOrEmpty(clientSecret) ? "[empty]" : "*****")}");
```

Comment:
Is it necessary to do print this?

Reply:
Changed it so that it only logs if url or client secret is missing